.azure/pipelines/ci.yml

```diff
@@ -394,6 +394,29 @@ jobs:
       path: artifacts/TestResults/
       publishOnError: true
 
+- template: jobs/default-build.yml
```

Comment:
This will run in every build, PR or not. Suggest creating a build matrix in the Test Windows Server 2016 x64 job and make the Debug side of it conditional on it _not_ being a PR build, then use `$(_BuildConfig)` instead of `Release` where necessary. 

Overall, this is kind of the opposite of what we do in https://github.com/aspnet/Extensions. Extensions code of interest is https://github.com/aspnet/Extensions/blob/8385543edac69a2b817756d1ba81a63acf76518d/azure-pipelines.yml#L78-L88.

In particular, I'm suggesting
``` yaml
strategy:
  matrix:
    ${{ if notin(variables['Build.Reason'], 'PullRequest') }}:
      Debug:
        _BuildConfig: Debug
    Release:
      _BuildConfig: Release
```

Could also only do the Debug side of the matrix in (say) internal builds. Up to you.

And, could add a schedule for this pipeline (avoiding something totally separate) and use `in(variables['Build.Reason'], 'PullRequest')` as the condition above to only build Debug daily (for example).